scripts/generate-app-config/generateEvidenceConfig.js

```diff
@@ -21,6 +21,11 @@ const getScreenshotPatterns = generatePagesOptions => (
       // build out a regex for the entrypoint mask.
       entryPoint: `${rootPath}/(test|tests)/wdio/__snapshots__/reference/`,
     });
+    acc.push({
+      pattern: `${rootPath}/{test,tests}/wdio/*/__snapshots__/reference/**/*.png`,
+      // build out a regex for the entrypoint mask.
+      entryPoint: `${rootPath}/(test|tests)/wdio/[^/]*/__snapshots__/reference/`,
```

Comment:
Would it be possible to update the regex of the existing regex patterns or do we need a new route to achieve this search pattern?

Reply:
Updated. Tested locally with and without subfolders.